dbt/adapters/databricks/python_submissions.py

```diff
@@ -288,7 +287,7 @@ def get_elapsed() -> float:
 
         while get_elapsed() < MAX_CLUSTER_START_TIME:
             status_response = self.get_cluster_status()
-            if status_response.get("state") == "Running":
+            if status_response.get("state") == "RUNNING":
```

Comment:
Since this behaviour previously worked, what changed?. Did DBR change the capitalisation? If so, we need to check for `.get("state") in ["Running", "RUNNING"]` or similar. Thoughts?

Reply:
Has the behavior previously worked?  Do we have evidence of that?

Reply:
I'm fine to make the comparison more flexible :)

Reply:
Yep, the evidence is in the PR that implemented this behaviour a couple months ago: https://github.com/databricks/dbt-databricks/pull/306

Our integration test used to pass.

This wouldn't be the first time DBR change capitalisation and broke things for us, though. We had the same problem with `pysql` a few months ago 🤦

Reply:
> I'm fine to make the comparison more flexible :)

I think we should, since applying this fix will break things for anyone using older DBRs.